src/main/java/core/selenium/utils/ReaderFile.java

```diff
@@ -0,0 +1,33 @@
+package core.selenium.utils;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * ReaderFile class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public class ReaderFile {
+
+    /**
+     * Gets Properties class with data.
+     *
+     * @return a Properties object.
+     */
+    public static Properties getProperties() {
+        Properties prop = new Properties();
+        try (InputStream input = new FileInputStream("gradle.properties");) {
+            prop.load(input);
+        } catch (FileNotFoundException ex) {
+            ex.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Why do you have many catch?

Reply:
First I use "catch (FileNotFoundException ex)" to sent an exception if don't find a file.
Second I use  "catch (IOException e)" to sent an exception if I have a problem with the body of try "prop.load(input)", if the "input" parameter is null.